go.mod

```diff
@@ -114,3 +114,5 @@ retract (
 	v0.3.0 // Published accidentally
 	v0.2.0 // Published accidentally
 )
+
+replace github.com/uber/cadence-idl => ./idls
```

Comment:
replaces do not work for anyone importing the client, so this only works for local developers